source/views/sis/student-work/detail.android.js

```diff
@@ -137,6 +138,14 @@ function LastUpdated({when}: {when: string}) {
 	) : null
 }
 
+function shareJob(job: JobType) {
```

Comment:
One more - can you pull shareJob into a separate file, please? I dislike having the exact same thing in two places.

Reply:
Done.